subprojects/integ-test/src/integTest/groovy/org/gradle/integtests/StaleOutputIntegrationTest.groovy

```diff
@@ -132,15 +132,10 @@ class StaleOutputIntegrationTest extends AbstractIntegrationSpec {
             gradle.version=1.0
         """
         and:
-        succeeds("help")
-        then:
-        // It looks like the build may have been run with a different version of Gradle
-        // The build output has been removed
-        javaClassFile("Main.class").assertDoesNotExist()
```

Comment:
This code is removed because we now only remove the stale output right before the execution of the task, right?